pkg/backup/restore/restore_manager.go

```diff
@@ -172,7 +172,9 @@ func (rm *restoreManager) syncRestoreJob(restore *v1alpha1.Restore) error {
 					return err
 				}
 
-				err = s.AddVolumeTags(pvs)
+				if restore.Spec.S3 != nil {
+					err = s.AddVolumeTags(pvs, restore.Spec.S3.Region)
```

Comment:
S3 supports cross region access. So maybe s3 region is not equal to volume region. I think maybe we can get region from the restoremeta file.

Reply:
Use another set of API to get EC2's region.